src/components/OrganizationsList/OrganizationsList.tsx

```diff
@@ -26,6 +30,25 @@ export function OrganizationsList({
   const orgsSorted = sortObjectsByParam(orgs, 'name');
   const orgsGrouped = chunkArray(orgsSorted, 3);
 
+  const getOrgRolesText = (orgGuid: string): React.ReactNode => {
+    const orgRoles = roles[orgGuid];
+    if (!orgRoles || !orgRoles.length) {
+      return (
+        <>
+          You're a <strong>User</strong> in this organization.
+        </>
+      );
+    }
+    const formattedRoles = orgRoles
+      .map<React.ReactNode>((r) => (
+        <span className="text-bold text-capitalize" key={`${orgGuid}-${r}`}>
+          {formatOrgRoleName(r).replace('org ', '')}
+        </span>
+      ))
+      .reduce((prev, cur) => [prev, ' and ', cur]);
+    return <>You're a {formattedRoles} in this organization.</>;
```

Comment:
It’s me, the terrible grammar goblin:

<img width="174" alt="image" src="https://github.com/user-attachments/assets/44a88f2b-72dd-4bc0-871a-0b6bd6416dad">

If possible, could this be `You’re a**n**` when `{formattedRoles}` begins with a vowel? (Also, stretch goal: could we make this “You’re” (proper curly quote, rather than straight prime)?)

Reply:
> Time for some generative AI

eleni